Jenkinsfile

```diff
@@ -150,47 +156,54 @@ pipeline {
           steps {
             script {
               def coverageEnabled = false
+              def cmakeOptions = ""
               if (!params.Linux) {
                 coverageEnabled = true
+                cmakeOptions = " -DCOVERAGE=ON "
               }
               def scmVars = checkout scm
               env.IROHA_VERSION = "0x${scmVars.GIT_COMMIT}"
               env.IROHA_HOME = "/opt/iroha"
               env.IROHA_BUILD = "${env.IROHA_HOME}/build"
 
               sh """
-                /usr/local/bin/ccache --version
-                /usr/local/bin/ccache --show-stats
-                /usr/local/bin/ccache --zero-stats
-                /usr/local/bin/ccache --max-size=5G
+                ccache --version
+                ccache --show-stats
+                ccache --zero-stats
+                ccache --max-size=5G
               """
               sh """
-                /usr/local/bin/cmake \
-                  -DCOVERAGE=ON \
+                cmake \
                   -DTESTING=ON \
                   -H. \
                   -Bbuild \
                   -DCMAKE_BUILD_TYPE=${params.BUILD_TYPE} \
-                  -DIROHA_VERSION=${env.IROHA_VERSION}
+                  -DIROHA_VERSION=${env.IROHA_VERSION} \
+                  ${cmakeOptions}
               """
-              sh "/usr/local/bin/cmake --build build -- -j${params.PARALLELISM}"
-              sh "/usr/local/bin/ccache --show-stats"
+              sh "cmake --build build -- -j${params.PARALLELISM}"
```

Comment:
Accessing `params.PARALLELISM` directly works just fine even in case of the initial build. However, if you assign this value to a local variable inside Groovy script and then use it for a build, it's value always null